src/schema/v2/schema.ts

```diff
@@ -115,6 +115,7 @@ const rootFields = {
   artworks: Artworks,
   artist: Artist,
   artists: Artists,
+  artistsByLetterConnection,
```

Comment:
what do you think if this was just `artistsConnection`? Encoding the `...ByLetter` part feels like it's specific to an argument, whereas a more general purpose `artistsConnection` (which can expand in the future if needed for other args) feels maybe better.

Reply:
👍